application/src/Form/SiteSettingsForm.php

```diff
@@ -124,6 +134,32 @@ public function init()
             ],
         ]);
 
+        $langFieldset->add([
+            'name' => 'filter_locale_values',
+            'type' => 'checkbox',
+            'options' => [
+                'label' => 'Filter values based on site localization', // @translate
```

Comment:
"Localization" is likely to get confusing here with a Language section and Locale setting, it should probably say "locale" here.

Reply:
OK updated in cccdf45